model/textparse/openmetricsparse.go

```diff
@@ -211,6 +213,11 @@ func (p *OpenMetricsParser) Exemplar(e *exemplar.Exemplar) bool {
 	return true
 }
 
+// CreatedTimestamp returns false because OpenMetricsParser does not support created timestamps.
```

Comment:
```suggestion
// CreatedTimestamp returns false because OpenMetricsParser does not support created timestamps (yet).
```

Reply:
Can you add quick issue about it?

Reply:
Here it is: https://github.com/prometheus/prometheus/issues/12980